raw/mdb/api.xml

```diff
@@ -188,7 +188,7 @@ is opaque to the walker: it must not modify or dereference this value, nor
 can it assume it is a pointer to valid memory.</para>
 <para>The starting address for the walk is stored in <replaceable>walk_addr</replaceable>.
 This is either NULL if <function>mdb_walk</function> was called, or the address
-parameter specified to <function>mdb_pwalk</function>. If the <command>::walk</command> built-in
+parameter specified to <function>mdb_pwalk</function> or <function>mdb_fpwalk</function>. If the <command>::walk</command> built-in
```

Comment:
Sorry for the long delay in getting to this; however, I don't think this integrated with the name mdb_fpwalk did it? It looks like this was meant to be mdb_fpwalk_dcmd? Is that correct here and below?